packages/alfa-rules/test/sia-er62/rule.spec.tsx

```diff
@@ -904,3 +904,44 @@ test(`evaluate() is inapplicable to an <a> element with a <p> parent element
 
   t.deepEqual(await evaluate(ER62, { document }), [inapplicable(ER62)]);
 });
+
+test(`evaluate() passes an <a> element that has a difference in contrast of 3:1 as a distinguishing feature`, async (t) => {
+  const target = <a href="#">Link</a>;
+
+  const document = h.document(
+    [<p>Hello {target}</p>],
+    [
+      h.sheet([
+        h.rule.style("p", {
+          color: "rgb(148, 148, 148)",
+        }),
+
+        h.rule.style("a", {
+          textDecoration: "none",
+          outline: "none",
+          cursor: "auto"
+        }),
+      ]),
+    ]
+  );
+
+  const style = Ok.of(
+    ComputedStyles.of([
+      ["border-width", "0px"],
+      ["font", "16px serif"],
+      ["color", "rgb(0% 0% 93.33333%)"],
+      ["outline", "0px"],
+    ])
+  );
+
+  t.deepEqual(await evaluate(ER62, { document }), [
+    passed(ER62, target, {
+      1: Outcomes.IsDistinguishable([style], [style], [style]),
+    }),
+  ]);
+});
+
+// Check gradient to non-gradient
+// Check non-gradient to gradient
+// Check gradient to gradient
```

Comment:
We already have (or should 🤞 ) tests that `getColor` behaves in presence of gradients, so we don't need to be too heavy here (is the new test going to detect bugs in the rule or in `getColor`?)
We still want one test with several pairings (especially once we have the extended diagnostic) to be sure that we do compute and send all of them correctly.

Reply:
I gave this one a shot, now the gradient specific tests should be in `get-colors.spec.tsx`